### PR TITLE
Turn on no implicite any flag

### PR DESCRIPTION
**Pull request by przemyslawjanpietrzak**

### CODE REVIEW

**Comment by przemyslawjanpietrzak** [comment]
@starspire my point was to make making more implicite any impossible and allow on-boarded devs to track any easily. Im not able to replace `any` with proper type and to be honest I don't understand what gardener do.

**Comment by kss-espeo** [comment]
@starspire @przemyslawjanpietrzak I could replace any with types. Will put these changes in this PR an re-submit it for review.

**Comment by przemyslawjanpietrzak** [comment]
@kss-espeo Great, you can push directly to this branch

File: src/application/selector/json/JsonResultConverter.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class JsonResultConverter {
-  static toString(results: any) {
+  static toString(results: any[]) {
```
Comment (przemyslawjanpietrzak) [inline src/application/selector/json/JsonResultConverter.ts]: try unknown[] instead of any[]

Comment (kss-espeo) [reply src/application/selector/json/JsonResultConverter.ts]: Tried, didn't work. `any` is what corresponding lib (`jsonpath`) returns so I'd like to leave it as-is.

File: src/application/dataFetcher/random/RandomSgxDataFetcherAdapter.ts [hunk 2]
```diff
@@ -20,8 +22,8 @@ class RandomSgxDataFetcherAdapter implements DataFetcher {
   }
 
   async fetch(request: Request) {
-    const min = RequestUrlParser.resolveLeftSideBound(request.url);
-    const max = RequestUrlParser.resolveRightSideBound(request.url);
+    const min: number = +RequestUrlParser.resolveLeftSideBound(request.url);
```
Comment (przemyslawjanpietrzak) [inline src/application/dataFetcher/random/RandomSgxDataFetcherAdapter.ts]: TS should know that min is number

File: src/application/dataFetcher/random/RandomSgxDataFetcherAdapter.ts [hunk 3]
```diff
@@ -20,8 +22,8 @@ class RandomSgxDataFetcherAdapter implements DataFetcher {
...
+    const max: number = +RequestUrlParser.resolveRightSideBound(request.url);
```
Comment (przemyslawjanpietrzak) [inline src/application/dataFetcher/random/RandomSgxDataFetcherAdapter.ts]: TS should know that min is number

File: src/application/dataFetcher/random/RandomSgxDataFetcherAdapter.ts (hunk 2 above)
Comment (kss-espeo) [reply src/application/dataFetcher/random/RandomSgxDataFetcherAdapter.ts]: do you mean like this? `const min = +RequestUrlParser.resolveLeftSideBound(request.url);`

**Pull request merged**